subsys/bluetooth/mesh/shell/shell_dm_cli.c

```diff
@@ -35,7 +35,7 @@ static int cmd_cfg(const struct shell *shell, size_t argc, char *argv[])
 		return err;
 	}
 
-	if (!mod && !shell_vnd_model_first_get(BT_MESH_VENDOR_COMPANY_ID,
```

Comment:
I believe this should not be changed. We want to allow people to update/expand this model and use it for their own purpose and in that case, substitute this number with their own company ID. Applies to all changes in this file.